plugin.json

```diff
@@ -21,6 +21,13 @@
     "settings_schema": {
         "header": "The GitHub plugin for Mattermost allows users to subscribe to notifications, stay up-to-date with reviews, see the status of pull requests at a glance, and other common GitHub actions - directly from Mattermost. \n \n Instructions for setup are [available here](https://www.mattermost.com/pl/default-github-plugin#configuration).",
         "settings": [
+            {
+                "key": "UsePreregisteredApplication",
+                "display_name": "Use Preregistered OAuth Application:",
```

Comment:
```suggestion
                "display_name": "Use the Mattermost Integration App listed in Github's Marketplace:",
```

Reply:
@aaronrothschild  Do we actually plan to list the app in Github Marketplace? I do not think there is actually any point in doing that as with Chimera customers will just authorize our existing OAuth app and it does not have anything to do with Marketplace. 

I might not be familiar with the whole vision, so could elaborate what is the goal here?

Reply:
Generally, we want customers to be aware that our solution can be integrated with other solutions.  It builds awareness of Mattermost as an option and if someone is using Mattermost when scanning the Marketplace, they are more likely to then integrate it.  I realize now that you are using the single user's account to generate the app programmatically in their environment, but if there is an App that CAN be listed in their Marketplace, I would like us to do that.